client/jetpack-cloud/sections/partner-portal/issue-license-form/style.scss

```diff
@@ -29,46 +29,57 @@
 }
 
 .issue-license-form__controls {
-	text-align: right;
-
-	@include break-large {
-		text-align: left;
+	position: fixed;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	padding: 0.875rem 0.875rem 1.5rem;
+	background: white;
+	box-shadow: 0 -1px 2px rgba( 0, 0, 0, 0.12 );
+
+	.button {
+		width: 100%;
+		padding: 8px 24px;
+		font-size: 1rem;
+		white-space: nowrap;
 	}
-}
 
-.issue-license-form__top {
-	margin-bottom: 1rem;
-	margin-top: -1.5rem;
-	display: block;
+	@include breakpoint-deprecated( '>660px' ) {
+		position: static;
+		padding: 0;
+		background: transparent;
+		box-shadow: none;
 
-	@include break-large {
-		display: flex;
-		justify-content: space-between;
+		.button {
+			width: auto;
+		}
 	}
+}
 
-	& > div {
-		align-self: flex-end;
-		margin-bottom: 0;
-	}
+.issue-license-form__top {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 0.5rem;
 }
 
 .issue-license-form__bottom {
 	display: flex;
 	flex-wrap: wrap;
 
-	@include break-wide {
+	@include break-xlarge {
 		margin-left: -0.5rem;
 		margin-right: -0.5rem;
 	}
 }
 
 p.issue-license-form__description {
-	margin: 2rem 0 1rem;
+	flex: 1 1 auto;
 	align-self: flex-end;
+	margin: 0;
 	font-size: 0.875rem;
 	color: #333;
 
-	@include break-large {
-		margin: 0;
+	@include breakpoint-deprecated( '>660px' ) {
```

Comment:
I'm guessing the choice of using `breakpoint-deprecated( '>660px' )` is because other components on the page does it; but we could use `@include break-small` as well

_Notice: `break-small` breaks at 600px so there's a 60px difference between the breakpoints_

Reply:
The key point is that the sidebar hides at 660 which completely changes the layout otherwise I would've avoided the deprecated breakpoint.